docs/api/_index.md

```diff
@@ -0,0 +1,749 @@
+---
+title: "HTTP API"
+linkTitle: "HTTP API"
+weight: 5
+slug: api
+menu:
+no_section_index_title: true
+---
+
+Cortex exposes an HTTP API for pushing and querying time series data, other then operating the cluster itself.
+
+For the sake of clarity, in this document we have grouped API endpoints by service, but keep in mind that they're exposed both when running Cortex in microservices and singly-binary mode:
+- **Microservices**: each service exposes its own endpoints
+- **Single-binary**: the Cortex process exposes all API endpoints for the services running internally
+
+## Endpoints
+
+| API | Service | Endpoint |
+| --- | ------- | -------- |
+| [Index page](#index-page) | _All services_ | `GET /` |
+| [Configuration](#configuration) | _All services_ | `GET /config` |
+| [Services status](#services-status) | _All services_ | `GET /services` |
+| [Readiness probe](#readiness-probe) | _All services_ | `GET /ready` |
+| [Remote write](#remote-write) | Distributor | `POST /api/v1/push` |
+| [Tenants stats](#tenants-stats) | Distributor | `GET /distributor/all_user_stats` |
+| [HA tracker status](#ha-tracker-status) | Distributor | `GET /distributor/ha_tracker` |
+| [Flush chunks / blocks](#flush-chunks--blocks) | Ingester | `GET|POST /ingester/flush` |
```

Comment:
<img width="1004" alt="Screen Shot 2020-08-19 at 9 22 52 AM" src="https://user-images.githubusercontent.com/6921077/90640122-99b56980-e1fd-11ea-9964-b2d6fc545400.png">

I'm not sure what is causing it. However, I notice weird formatting in the rich text diff starting at this point.

Reply:
I think this is because the pipe character is not escaped as Wing pointed out below

Reply:
This looks a to a GitHub issue. If I build the website (I previewed it locally) it renders correctly. On the contrary, if I escape it with `\|` the markdown parser used by the website builder keeps it as `\|` which is not what we want.

Reply:
Example of website rendering:

![Screenshot 2020-08-19 at 16 51 45](https://user-images.githubusercontent.com/1701904/90650528-48789a80-e23c-11ea-8af9-32ae479dbc2f.png)

Reply:
How about change `|` to `, `?

Reply:
I would do it only if it turns out that the GitHub markdown rendering breaks once merged. I'm not sure if it's just the markdown diff preview we see in the PR.

Reply:
Yes, I had to use the `,` instead of `|`. Thanks for the suggestion!
https://github.com/cortexproject/cortex/pull/3081